p-token/src/entrypoint-runtime-verification.rs

```diff
@@ -2773,13 +2847,16 @@ fn test_process_mint_to_checked(accounts: &[AccountInfo; 3], instruction_data: &
     } else if accounts.len() < 3 {
         assert_eq!(result, Err(ProgramError::NotEnoughAccountKeys));
         return result;
-    } else if accounts[1].data_len() != Account::LEN {
+    } else if accounts[1].data_len() != Account::LEN { // TODO Daniel: is it possible for something to be provided that has the same len but is not an account?
```

Comment:
The cheat codes currently assert that `data_len` is correct for the respective type of thing.
Account: 165, Mint: 82, Rent: 17